controllers/cron_rollout_helpers.go

```diff
@@ -174,41 +175,36 @@ func CreateDisruptionFromTemplate(ctx context.Context, cl client.Client, scheme
 }
 
 // getScheduledTimeForDisruption returns the scheduled time for a particular disruption.
-func getScheduledTimeForDisruption(disruption *chaosv1beta1.Disruption) (*time.Time, error) {
+func getScheduledTimeForDisruption(log *zap.SugaredLogger, disruption *chaosv1beta1.Disruption) time.Time {
```

Comment:
why don't we return the error anymore?

Reply:
this is a private method, and in the caller (Reconcile loop), we already checked for cron expression validity earlier. Hence imho, this method can now assume a valid cron description. Its no longer meaningful to check again, and just makes code longer unnecessarily.

https://github.com/DataDog/chaos-controller/blob/b28cfba90cb7a6fa7bfa9571143859fc8c4989a1/controllers/disruption_cron_controller.go#L249